client/a8c-for-agencies/sections/marketplace/products-overview-v2/hocs/with-product-lightbox.tsx

```diff
@@ -0,0 +1,139 @@
+import { useTranslate } from 'i18n-calypso';
```

Comment:
I am moving the product lightbox logic into a reusable HOC, as I intend to apply this on the Overview page card.